From 4b03e1294036b92118554119d23bf8731247cc93 Mon Sep 17 00:00:00 2001
From: Michael Ellerman <mpe@ellerman.id.au>
Date: Thu, 11 Jan 2018 16:32:16 +1100
Subject: [PATCH 1/6] powerpc: Mark CONFIG_PPC_DEBUG_RFI as BROKEN
Patch-mainline: Submitted, 2018-01-09 LKML + linuxppc-dev
References: bsc#1075088

This is debug code that doesn't work in all situations, mark it as
BROKEN to avoid confusion.

Signed-off-by: Michael Ellerman <mpe@ellerman.id.au>
Reviewed-by: Torsten Duwe <duwe@suse.de>

---
 arch/powerpc/Kconfig.debug | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/arch/powerpc/Kconfig.debug b/arch/powerpc/Kconfig.debug
index 67959c1002af..f6252f5e61d0 100644
--- a/arch/powerpc/Kconfig.debug
+++ b/arch/powerpc/Kconfig.debug
@@ -285,7 +285,7 @@ config PPC_EARLY_DEBUG_CPM_ADDR
 
 config PPC_DEBUG_RFI
 	bool "Debug RFIs (Return From Interrupt)"
-	depends on PPC_BOOK3S_64
+	depends on PPC_BOOK3S_64 && BROKEN
 	help
 	  The enables extra debug code in some RFI (Return From Interrupt)
 	  sequences, to detect that we are returning to the correct context
-- 
2.14.3

